assets/panel/prefs.js

```diff
@@ -1,16 +1,11 @@
 /* Any copyright is dedicated to the Public Domain.
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
-#ifdef RELEASE_OR_BETA
-pref("devtools.debugger.new-debugger-frontend", false);
-#else
 pref("devtools.debugger.new-debugger-frontend", true);
-#endif
```

Comment:
i'm pretty sure this will fail in mochitests

Reply:
but lets see...

Reply:
What do you mean? Do you run old the debugger mochitests on CI?

Reply:
I see
> ./mach mochitest --subsuite devtools --setenv MOZ_HEADLESS=1 devtools/client/debugger/new/test/mochitest/

I don't think this pref should affect any of them :/

Reply:
nope, it's a build issue... it needs to have some #if block or whatever to be considered 

we need the moz.build to be this:
```
# -*- Mode: python; indent-tabs-mode: nil; tab-width: 40 -*-
# vim: set filetype=python:
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.

JS_PREFERENCE_PP_FILES += [
    'devtools.js'
]

JS_PREFERENCE_FILES += [
    'debugger.js',
]

```

but it's 

```
# -*- Mode: python; indent-tabs-mode: nil; tab-width: 40 -*-
# vim: set filetype=python:
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.

JS_PREFERENCE_PP_FILES += [
    'devtools.js'
    'debugger.js'
]

```

we could replace the moz.build with our own like we're replacing other files as well....